journal-letters/modernized/05-chapter-3.adoc

```diff
@@ -328,11 +328,16 @@ After this meeting we went again to Stockport,
 and lodged at George Jones`'s. Here he wrote a letter
```

Comment:
This one needs to be dropped down onto its own line, then offset, bracketed, and kill the long dash.